songbank-backend/backend.js

```diff
@@ -35,55 +35,55 @@ var spotify_client_secret = process.env.SPOTIFY_CLIENT_SECRET;
 var spotify_redirect_uri = 'http://localhost:3000/auth/callback';
 
 var generateRandomString = function (length) {
-  var text = '';
-  var possible = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
+    var text = '';
+    var possible = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
 
-  for (var i = 0; i < length; i++) {
-    text += possible.charAt(Math.floor(Math.random() * possible.length));
-  }
-  return text;
+    for (var i = 0; i < length; i++) {
+        text += possible.charAt(Math.floor(Math.random() * possible.length));
+    }
+    return text;
 };
 
 app.get('/auth/login', (req, res) => {
 
-  var scope = "streaming user-read-email user-read-private";
-  var state = generateRandomString(16);
-
-  var auth_query_parameters = new URLSearchParams({
-    response_type: "code",
-    client_id: spotify_client_id,
-    scope: scope,
-    redirect_uri: spotify_redirect_uri,
-    state: state
-  })
-  console.log(spotify_client_id);
-  res.redirect('https://accounts.spotify.com/authorize/?' + auth_query_parameters.toString());
+    var scope = "streaming user-read-email user-read-private";
+    var state = generateRandomString(16);
+
+    var auth_query_parameters = new URLSearchParams({
+        response_type: "code",
+        client_id: spotify_client_id,
+        scope: scope,
+        redirect_uri: spotify_redirect_uri,
+        state: state
+    })
+    console.log(spotify_client_id);
+    res.redirect('https://accounts.spotify.com/authorize/?' + auth_query_parameters.toString());

```

Comment:
Good changes to spacing it looks much cleaner

Reply:
Thank you